<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceQueryFaultPage</class>
 <widget class="QWidget" name="MaintainceQueryFaultPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(0,0,50);
}QPushButton
{
font: 20px, &quot;微软雅黑&quot;;
color: black;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(240,240,240,255),stop:1 rgba(50,50,50,255));
}
QLabel
{
	color:white;
	font: 20px, &quot;微软雅黑&quot;;
}</string>
  </property>
  <widget class="QPushButton" name="BTN4">
   <property name="geometry">
    <rect>
     <x>316</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN8">
   <property name="geometry">
    <rect>
     <x>724</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string>返  回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHome">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>620</y>
     <width>91</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/image/Home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN2">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: black;transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN5">
   <property name="geometry">
    <rect>
     <x>418</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN7">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>826</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN3">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>620</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px, &quot;微软雅黑&quot;;
    color: black;
	transparent;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 rgba(170,170,170,255),stop:1 rgba(150,150,150,255));
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTC2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>TC2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNMP2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>MP2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNM2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px;
</string>
   </property>
   <property name="text">
    <string>M2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNM1">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>M1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNMP1">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>MP1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTC1">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>30</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>TC1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>位置查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNHVAC">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>HVAC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNDCU">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>210</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>DCU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTCMS">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>TCMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNBCU">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>BCU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNACU">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px;
</string>
   </property>
   <property name="text">
    <string>ACU</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNPIS">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>PIS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>设备查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNREC">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>210</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px;
</string>
   </property>
   <property name="text">
    <string>REC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNEDCU">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>210</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px;
</string>
   </property>
   <property name="text">
    <string>EDCU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>代码查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBLCodeInput">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px  solid gray;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNQUERY">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>410</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum9">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum0">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>550</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNumC">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>51</width>
     <height>101</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNNum5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>490</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>有故障</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>550</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>无故障</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNS1">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>490</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px;
</string>
   </property>
   <property name="text">
    <string>TCMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNS2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>550</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 lightgray,stop:1 rgba(0,0,50,255));border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px ;
</string>
   </property>
   <property name="text">
    <string>TCMS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNTRAIN">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 0, 50);border:2px groove  rgb(100, 100, 200);border-radius:10px;padding:2px 4px</string>
   </property>
   <property name="text">
    <string>TRAIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Alarm">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>410</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:16px;
color:red;</string>
   </property>
   <property name="text">
    <string>历史故障中无此代码故障!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
